pkg/trace/config/config.go

```diff
@@ -595,6 +596,14 @@ func noopContainerTagsFunc(_ string) ([]string, error) {
 	return nil, ErrContainerTagsFuncNotDefined
 }
 
+// ErrContainerIDFromOriginInfoFuncNotDefined is returned when the ContainerIDFromOriginInfo function is not defined.
+var ErrContainerIDFromOriginInfoFuncNotDefined = errors.New("ContainerIDFromOriginInfo function not defined")
```

Comment:
nit: these long names are getting increasingly harder to read. Consider using shorter names or even having a dedicated type for Container ID generation errors.

Not a dealbreaker though, just something for consideration. Leaving the PR approved.